src/components/evaluate.ts

```diff
@@ -12,8 +11,7 @@ export function evaluate(str: string, context: any, currentProp: string, fallbac
     // Only consider properties ABOVE the current property in the schema
     // (This enforces a top-to-bottom data dependency which is just nice.)
     if (prop === currentProp) break;
-    const short = shortName(prop);
-    _context[short] = context[prop];
+    _context[prop] = context[prop];
```

Comment:
I moved the `shortName`-ing logic out of this function and into the caller, so that `area` can be used to override the default shortname generated by this logic.